system/lib/gl/webgl1.c

```diff
@@ -27,8 +28,19 @@ static void InitWebGLTls() {
   pthread_key_create(&currentThreadOwnsItsWebGLContext, NULL);
 }
 
-static pthread_t GetCurrentTargetThread() {
-  return *(void**)(pthread_getspecific(currentActiveWebGLContext) + 4);
+// When OFFSCREEN_FRAMEBUFFER is enabled the EMSCRIPTEN_WEBGL_CONTEXT_HANDLE
+// is a struct with two fields.  See registerContext in library_webgl.js
```

Comment:
(The JS side looks like it needs updates for memory64, which I guess you might already have in another PR?)

Reply:
Yes, that is what the other PR going to be.